ui/v2.5/src/locales/en-GB.json

```diff
@@ -151,7 +151,7 @@
       "interface": "Interface",
       "logs": "Logs",
       "plugins": "Plugins",
-      "scrapers": "Scrapers",
+      "scraping": "Scraping",
```

Comment:
Please remove `scrapers` from the other locale files.

Reply:
As internationalization is new I'm wondering how you would want to handle this. Do you want me to remove these lines from the other locale files, or change them into `"scraping": ""`? So updating the key, and making the value an empty string.
Adding (or updating) translations using an empty string IMO has the advantage that translators can find missing translations more easily and all files could be kept identical in format (so every file would have `"scraping"` on line 154 for example) which I think would make it easier to "compare" the files.

Reply:
This way would need some modification to our message loading code. Currently, we merge the chosen locale messages with the default message locale (`en-GB`), using lodash's `merge` function. This combines the chosen and default maps, using the default value if the chosen value resolves to `undefined`. You will probably need to use `mergeWith` supplying a customiser function to return the default value if the chosen value is empty.

I'm also not completely sold on this approach as it makes the `en-US` locale unnecessarily verbose (most of the messages should use the `en-GB` value). I'd prefer not to have to maintain identical locale file structures as well.

However, when adding new messages to add a new placeholder message to the other locales, I think it's not a bad approach to add an empty string placeholder value - provided the above code change is made.

Reply:
So if I do understand you correctly you _do_ want me to set the value / translation to `""` and update the translation loading code to ignore these values?

Reply:
Sorry, was offering an opinion rather than a directive. :)

If you're happy to do the changes to the message loading code, then go ahead and do it. Otherwise, just remove `scrapers` from the other json locale files.

Reply:
That clears it up, I will make the changes then.

I was a bit unsure because the first part of the comment seemed to imply you were totally against using `""` and at the end you seemed to find it a good solution (if the fix to merging is added as well, obviously).

Reply:
Yeah I can understand why it was confusing. What I wasn't keen on was trying to maintain identical file structures for all of the locale files.